lib/standards/aria-roles.js

```diff
@@ -117,8 +117,8 @@ const ariaRoles = {
   },
   combobox: {
     type: 'composite',
-    requiredOwned: ['listbox', 'tree', 'grid', 'dialog', 'textbox'],
-    requiredAttrs: ['aria-expanded'],
+    requiredOwned: ['listbox', 'tree', 'grid', 'dialog'],
+    requiredAttrs: ['aria-expanded', 'aria-controls', 'aria-owns'],
```

Comment:
This implies that both are required when in reality it's just one or the other. That's not something we can support, so maybe we just list the 1.2 spec required attrs and then special case `aria-owns` to support the 1.0/1 pattern in aria-required-attrs-evaluate.

Also add `aria-owns` to the `allowedAttrs` array and remove `aria-controls` from it (since it's now in the required array).

```suggestion
    requiredAttrs: ['aria-expanded', 'aria-controls'],
```